src/aznfswatchdog

```diff
@@ -129,6 +129,17 @@ reconcile_conntrack()
     done
 }
 
+# Detect and log distro and bash version
+if [ -r /etc/os-release ]; then
```

Comment:
This file is usually not present in centos. Please follow the way it is done in aznfs_install.sh as previously suggested.